src/algebra/big_operators/enat.lean

```diff
@@ -0,0 +1,22 @@
+/-
```

Comment:
Add this to `algebra/big_operators/default.lean`.

Reply:
Do we generally put theorems in `default` files? I think we shouldn't do that too much...

Reply:
Oh wait, you mean import the file, not adding the result there. Good point.

Reply:
My thought is that someone importing `algebra.big_operators` probably expects to get everything in that directory. This shouldn't affect mathlib, since we try to minimize imports and thus don't usually import any `default` files.

I don't think we've been consistent about this though, so no big deal.

Reply:
Related: #4588